test/externalTests/gp2.sh

```diff
@@ -67,11 +67,11 @@ function gp2_test
     force_hardhat_compiler_binary "$config_file" "$BINARY_TYPE" "$BINARY_PATH"
     force_hardhat_compiler_settings "$config_file" "$(first_word "$SELECTED_PRESETS")" "$config_var"
     force_hardhat_unlimited_contract_size "$config_file" "$config_var"
-    npm install
+    yarn
```

Comment:
> The [gp2 repo](https://github.com/gnosis/gp-v2-contracts) only contains a yarn.lock file and npm versions v7+ should support yarn.lock files (https://blog.npmjs.org/post/621733939456933888/npm-v7-series-why-keep-package-lockjson). However, this does not seem to be the case for our CI even though the npm version is 8.1.2.

Note that we're deliberately deleting `package-lock.json` and `yarn.lock` because we want the newest packages installed, since older versions do not always support newer compiler features. So the use of `yarn` vs `npm` should not matter here. Or at least not for this reason.

Reply:
Right, I was not aware of such removal. So the use of yarn or npm should not matter indeed. Not sure why using yarn fixed it though.

Reply:
Hmmm... maybe yarn just handles the missing typings differently. If that's the case then the workaround is fine. We don't care for `keyv` so it switching to another package manager makes it install properly, it's good enough for us.

Reply:
I honestly couldn't find a proper explanation for why the workaround works since, as you pointed out, we delete the lock files before installation.